src/test/java/winstone/testCase/HttpConnectorTest.java

```diff
@@ -129,12 +93,14 @@ public void testWriteAfterServlet() throws IOException,
         Launcher winstone = new Launcher(args);
 
         // Check for a simple connection
-        WebConversation wc = new WebConversation();
-        WebRequest wreq = new GetMethodWebRequest(
-                "http://localhost:10005/examples/TestWriteAfterServlet");
-        WebResponse wresp1 = wc.getResponse(wreq);
-        Logger.logDirectMessage(Logger.INFO, "log", "Output: " + wresp1.getText(), null);
-        assertTrue(wresp1.getText().endsWith("Hello"));
+        HttpRequest request =
+                HttpRequest.newBuilder(new URI("http://localhost:10005/examples/TestWriteAfterServlet")).GET().build();
```

Comment:
while you're doing some cleaning. maybe using a random port here would be good as well.
same as done in ArgumentsRealmTest

Reply:
Not interested